benchmark/common.js

```diff
@@ -25,6 +25,8 @@ function Benchmark(fn, configs, options) {
   if (options && options.flags) {
     this.flags = this.flags.concat(options.flags);
   }
+  if (process.env.NODE_BENCHMARK_FLAGS)
+    this.flags = this.flags.concat(process.env.NODE_BENCHMARK_FLAGS.split(' '));
```

Comment:
Maybe split on `/\s+/`? Otherwise `env NODE_BENCHMARK_FLAGS="--foo  --bar" node bench.js` (note the double space) produces three flags, one of them the empty string.